src/main/resources/META-INF/plugin.xml

```diff
@@ -163,13 +169,13 @@
         </action>
         <action id="cody.cycleForwardAutocompleteAction"
                 class="com.sourcegraph.cody.autocomplete.action.CycleForwardAutocompleteAction">
-            <keyboard-shortcut first-keystroke="alt OPEN_BRACKET"
-                               keymap="$default"/>
+            <keyboard-shortcut first-keystroke="alt OPEN_BRACKET" keymap="$default"/>
```

Comment:
Minor thing, but I think we should add `replace-all="true"` there as in other places. 
That ensures that if we ever change the binding in the plugin configuration old one will be removed.